### PR TITLE
Custom Attribute Filter(s) documentation

### PR DESCRIPTION
**Pull request by ivanmjartan**
---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10022/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11982/ : FAILURE in 4m 29s

File: libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx [hunk 1]
```diff
@@ -12,28 +12,94 @@ export const ALIGN_POINT = [
 ];
 
 /**
- * @alpha
+ * This is interface AttributeFilter dropdown button.
```
Comment (no23reason) [inline libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx:15]: Nitpick: interface OF

File: libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx [hunk 2]
```diff
@@ -12,28 +12,94 @@ export const ALIGN_POINT = [
...
 
+    /**
+     * This prop define open or close state of AttributeFilter dropdown
+     *
+     * @beta
+     */
     isOpen?: boolean;
 
+    /**
+     * This prop mean that AttributeFilter is initializing Attribute elements and its internal data
```
Comment (no23reason) [inline libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx:57]: This prop meanS (repeated several times here).

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10025/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11989/ : FAILURE in 4m 56s

File: libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx [hunk 3]
```diff
@@ -12,28 +12,94 @@ export const ALIGN_POINT = [
...
+     * Callback to open or close AttributeFilter dropdown
+     *
+     * @beta
+     */
     onClick?: () => void;
 }
 
 /**
- * @internal
+ * This component implement {@link IAttributeFilterDropdownButtonProps}
```
Comment (no23reason) [inline libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx:100]: That is not strictly true, the component uses that interface for its props. I would focus on what this component does. The prop type should be linked by TSDoc automatically.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10028/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11992/ : FAILURE in 4m 28s

File: libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx [hunk 4]
```diff
@@ -12,28 +12,94 @@ export const ALIGN_POINT = [
 ];
 
 /**
- * @alpha
+ * This is interface AttributeFilter dropdown button.
+ * It is opening Attribute filter dropdown and display title or subtitle, selection details and attribute filter statuses like loading or filtering.
```
[the remark this answers is not in the document]
Comment (ivanmjartan) [reply libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx]: done

File: libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx (hunk 2 above)
Comment (ivanmjartan) [reply libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx]: fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10029/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11996/ : SUCCESS in 19m 37s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10051/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/12023/ : SUCCESS in 17m 44s

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10059/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/12031/ : SUCCESS in 16m 03s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10064/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/12036/ : FAILURE in 3m 58s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10065/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/12042/ : SUCCESS in 17m 50s

File: libs/sdk-ui-kit/src/List/InvertableSelect/InvertableSelect.tsx [hunk 5]
```diff
@@ -15,13 +15,44 @@ import { ErrorComponent } from "@gooddata/sdk-ui";
 import { InvertableSelectItem } from "./InvertableSelectItem";
 
 /**
- * @internal
+ * Properties of List item component implementation
+ *
+ * @beta
```
Comment (kandl) [inline libs/sdk-ui-kit/src/List/InvertableSelect/InvertableSelect.tsx:20]: All things in sdk-ui-kit should be marked as `@internal`

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2912/ : FAILURE in 2m 05s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3217/ : SUCCESS in 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2059/ : SUCCESS in 22m 02s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/12058/ : SUCCESS in 16m 15s

**Pull request merged**